composer.json

```diff
@@ -10,20 +10,20 @@
     "homepage": "https://github.com/selective-php/basepath",
     "license": "MIT",
     "require": {
-        "php": "^7.1",
+        "php": "^7.2",
```

Comment:
According to the [Slim 4 framework doc](http://www.slimframework.com/docs/v4/start/installation.html), it seems to require `php-7.2` version at least.

This package is for `Slim` framework, and it also requires `php-7.2` version at least.